app/views/publish/courses/course_information/edit.html.erb

```diff
@@ -115,11 +121,21 @@
       <% end %>
 
       <%= f.govuk_text_area(:how_school_placements_work,
-        label: { text: @course.placements_heading, size: "s" },
+        value: @course.enrichments.last[:how_school_placements_work],
+        label: { text: "How school placements work", size: "s" },
         max_words: 350,
         rows: 15) %>
 
       <%= f.govuk_submit "Save" %>
     <% end %>
   </div>
+  <aside class="govuk-grid-column-one-third" data-qa="course__related_sidebar">
```

Comment:
I think we need to add this to nearly every sub section in the courses show page. Eg every one of these sub sections with change links allow you to copy info from another course. Might be worth double checking and adding the partial in the remaining places? 
 
<img width="739" alt="Screenshot 2022-03-28 at 09 58 42" src="https://user-images.githubusercontent.com/616080/160363423-c9bde3d1-22b3-4178-be75-cecd18034e96.png">